.cirrus.yml

```diff
@@ -646,6 +736,40 @@ local_system_test_task: &local_system_test_task
     always: *logs_artifacts
 
 
+# Always run subsequent to integration tests.  While parallelism is lost
+# with runtime, debugging system-test failures can be more challenging
+# for some golang developers.  Otherwise the following tasks run across
+# the same matrix as the integration-tests (above).
+local_system_test_aarch64_task: &local_system_test_task_aarch64
+    name: &std_name_fmt_aarch64 "$TEST_FLAVOR $PODBIN_NAME $DISTRO_NV $PRIV_NAME $TEST_ENVIRON"
```

Comment:
No need to define an ARM-specific anchor for this (since the actual content is identical).   Unless, were you intending to include "(ARM64)" in the string?

Reply:
Minor nit: The `std_name_fmt_aarch64` anchor isn't referenced anywhere, you can remove it.  Also, I think since you're defining `DISTRO_NV` to include "aarch64", I think you can just use the `*std_name_fmt` alias here, no need to redefine.

Reply:
done.